grafana/Dockerfile

```diff
@@ -1,13 +1,64 @@
+ARG GRAFANA_VERSION=7.5.16
+ARG GOLANG_VERSION=1.19.0-alpine3.16
```

Comment:
please just hardcode it to 1.19-alpine3.16
It showed that this is flexible enough